openpype/hosts/blender/api/lib.py

```diff
@@ -274,6 +270,14 @@ def update_scene_containers():
             container_metadata.get("name"),
         )
 
+        # Check if container already exists
+        if (
+            container_name in bpy.context.window_manager.openpype_containers
+            and container_name not in entity.name
+        ):
+            add_datablocks_to_container({entity} | datablocks, container)
```

Comment:
I tried to cherry-pick this commit into our studio branch, and `container` is still undefined. I manually checked occurrences of `container` in case my LSP was acting silly, and no the LSP's right. There is no reference to `container` before this line.